quickwit-backward-compat/README.md

```diff
@@ -3,7 +3,50 @@
 This is just a project used to test backward compatibility of Quickwit.
 Right now, only SplitMetadata is tested.
 
-The build.rs script autocreates json sample files when something changes in the format.
-A unit test then checks that all of these sample files deserialize to the right
-`SplitMetadataAndFooterOffsets` object.
+We want to be able to read all past version of SplitMetadata, but only write the most recent format.
+The tests consists in pairs of json file.
+XXXX.json and XXXX.expected.json
+
+XXXX.json consists in a json file in a possibly old format.
```

Comment:
more details about "possibly" would be good

Reply:
```suggestion
XXXX.json consists in a json file in an old format.
```